vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -1364,8 +1364,11 @@ int get_sibling_index(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGr
    if (parent == INVALID_CELLID) {
       return ERROR;
    }
-   
-   vector<CellID> siblings = mpiGrid.get_all_children(parent);
+
+   // get_all_children returns an array instead of a vector now, need to map it to a vector for find and distance
+   std::array<uint64_t, 8> siblingarr = mpiGrid.mapping.get_all_children(parent);
+   vector<CellID> siblings(&siblingarr[0], &siblingarr[0] + sizeof(siblingarr) / sizeof(siblingarr[0]));
```

Comment:
That took me a moment to parse. Wouldn't ```siblangarr.begin(), siblingarr.end()``` fulfill the same function and be more understandable?

Reply:
Well, perhaps &siblingarr[0] can be replaced with siblingarr.begin()
but I tried
    vector<CellID> siblings(siblingarr.begin(),siblingarr.end());
and it gives major diffs. All the examples of generating a vector from an array used the method I used, apparently for a good reason.

Reply:
replacing &siblingarr[0] with siblingarr.begin() also gave diffs.

Reply:
> Little birds confirm this will be stable, good.
> But: should there not be some more elaborate AMR test, as this is used only in AMR code (except the () addition)?

Yeah, AMR testing underway and causing funny stuff. Don't merge yet.

Reply:
Bizarre. I guess my C++ kungfoo is weak today. Well, leave it as it is then.